MigrationTutorial/Models/V3/MachinaryAndTool.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using MongoDB.Bson;
+using Realms;
+
+namespace MigrationTutorial.Models.V3
+{
+    public class MachinaryAndTool : RealmObject
+    {
+        [PrimaryKey]
+        public ObjectId Id { get; private set; } = ObjectId.GenerateNewId();
+
+        private string _Type { get; set; }
+
+        public Type Type
+        {
+            get => Enum.Parse<Type>(_Type);
+            set => _Type = value.ToString();
+        }
+
+        private string _Status { get; set; }
+
+        public OperationalStatus Status
+        {
+            get => Enum.Parse<OperationalStatus>(_Status);
+            set => _Status = value.ToString();
+        }
+
+        public Employee? AssignedMaintaner { get; set; }
+
+        public string Brand { get; set; }
+
+        public string ToolName { get; set; }
+
+        public Supplier Supplier { get; set; }
+
+        public MachinaryAndTool() { }
+    }
+
+    public enum OperationalStatus
+    {
+        Malfunctioning = 1,
```

Comment:
Is there any particular reason we are specifying the numerical value?

Reply:
I've always done this as good practice. Being specific of the used default always seems a good idea to me.
Do you have anything against it?

Reply:
By default I think it starts with 0, and if you don't have anything against it starting from 0 (or giving specific values to all others), I'd personally keep it blank.